.github/workflows/test.yaml

```diff
@@ -73,17 +73,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python 3.11
-        uses: actions/setup-python@v5
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
         with:
-          python-version: "3.11"
-          cache: "pip"
-          cache-dependency-path: "**/pyproject.toml"
-      - name: set python default branch
+          cache-dependency-glob: pyproject.toml
+      - name: Install Hatch
+        run: uv tool install hatch
```

Comment:
Not really important, but shall we save two more lines by using `uvx hatch test` below?

Reply:
we can totally do that